GetConfigFiles.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
+import os
+import sys
+import argparse
+import json
+import pprint
+import mysql.connector
+
+
+def exec_sql(cursor, sql_stmt, params, error_msg):
+    """executes a simple sql statement"""
+    try:
+        cursor.execute(sql_stmt, params)
+    except Exception as err:
+        print(f"{str(err)} \n {error_msg}")
```

Comment:
What is the purpose of passing in an error message `error_msg` to these functions executing sql statements? And then we print the error that we passed? Sorry, not sure if I follow.

Reply:
@naved001 these helper functions are all an attempt to avoid a boilerplate `try/except` block around database calls. So e.g., you can `exec_sql(cursor, "SELECT name FROM users WHERE id=?", uid, "Failed to find user")` and then later you can `exec_sql(cursor, "SELECT color FROM widgets WHERE widgetid=?", widgetid, "Failed to find widget")`, and in both cases get a contextually appropriate error message printed to the console.

...but I'm not sure these helper functions are actually necessary or useful; see my comment later on.